src/sentry/relay/config/__init__.py

```diff
@@ -410,7 +410,7 @@ def _filter_option_to_config_setting(flt, setting):
         "d:transactions/measurements.lcp@millisecond",
         "d:transactions/measurements.app_start_cold@millisecond",
         "d:transactions/measurements.app_start_warm@millisecond",
-        "d:transactions/measurements.cls@millisecond",
+        "d:transactions/measurements.cls@none",
```

Comment:
Unfortunately we cannot create a static string in the indexer for this, because `d:transactions/measurements.cls@none` was already added for multiple organizations as a "custom" measurement (we allow up to five custom measurements per transaction, and this was treated as such because of the different unit).